base-containers/kvm/Dockerfile

```diff
@@ -0,0 +1,83 @@
+ARG   	VERSION=latest
+ARG	REGISTRY
+
+# Rockylinux does not enable 9P virtfs in the shipped QEMU builds, hence we have to rebuild our own version
+# Inspired from https://github.com/acudovs/qemu-kvm-virtfs/blob/master/rpmbuild/build
+FROM rockylinux:8 as builder
+
+# Enable additional repos
+RUN 	dnf -y install 'dnf-command(config-manager)' &&\
+   	dnf config-manager --set-enabled powertools
+
+# Install dependencies to build QEMU
+RUN 	yum -y update && yum -y install \
+        glusterfs-api-devel \
+        glusterfs-devel \
+        iasl \
+        libcacard-devel \
+	libpmem-devel \
+	nss-devel \
+	pkgconfig \
+	spice-protocol \
+	spice-server-devel \
+	usbredir-devel \
+	yum-utils \
+	'pkgconfig(epoxy)' \
+	'pkgconfig(gbm)' \
+	'pkgconfig(libdrm)'
+
+# Apply patches and build QEMU
+WORKDIR /opt
+RUN 	yum-builddep -y qemu-kvm
+RUN 	yumdownloader --source qemu-kvm
```

Comment:
As you are providing patches, it'll be more robust to hardcode the src rpm version/link. Otherwise at list the spec file will conflict once the maintainers push an new patch.

Reply:
This fix was on my todo list for this PR :)

Reply:
Fixed in https://github.com/UCL-INGI/INGInious/pull/949/commits/703ba30cf2c2d4d6d6b2dfdf9fb27e55c8045e28